src/Colliders.Core/Core.Colliders.Controller.cs

```diff
@@ -380,6 +395,16 @@ private void UpdateLegCollidersSkirtDB(DynamicBone dynamicBone)
                 }
             }
 #endif
+
+            /// <summary>
+            /// Get all active controllers
+            /// </summary>
+            static public IEnumerable<ColliderController> GetAllColliderControllers()
+            {
+                foreach (var controller in allControllers)
+                    if (controller && controller.enabled)
```

Comment:
To match the behaviour of `FindObjectsOfType` it would need to be `activeInHierarchy` instead of `enabled`, since it only cares about the GameObject being active in hierarchy. If this is not an issue then it would be fine as it is however.